### ISSUE TITLE
Conflict between wp-cli required in composer and platform.app.yaml

### ISSUE THREAD

**Issue by gilzow** [platformsh-templates/wordpress-composer#13]
**TL;DR** - update .platform.app.yaml to require the same version of `wp-cli/wp-cli-bundle` as composer.json instead of `wp-cli/wp-cli`. 

In [platform.app.yaml](https://github.com/platformsh-templates/wordpress-composer/blob/master/.platform.app.yaml#L17), we list wp-cli (just the core framework package) as a dependency.  

In [composer.json](https://github.com/platformsh-templates/wordpress-composer/blob/master/composer.json#L30) we require the wp-cli-bundle (the full cli tool, not just the framework). 

The instance of wp-cli required in .platform.app.yaml is installed to `/app/.global/wp-cli/wp-cli/` and a link in `/app/.global/vendor/bin`. The instance required in composer.json is installed to `/app/vendor/wp-cli` with a link in `/app/vendor/bin`.  Since `/app/.global/vendor/bin` is added to `$PATH`, this instance of wp-cli that is called with `wp` is the one in `.global`.  

[In wp-config.php](https://github.com/platformsh-templates/wordpress-composer/blob/master/wp-config.php#L5) we require the composer autoloader from the local composer install. We've now loaded two different autoloader files with potentially different versions of dependencies.
